cli/dcoscli/analytics.py

```diff
@@ -261,10 +262,18 @@ def _base_properties(conf=None):
         cmd = 'dcos'
         full_cmd = 'dcos'
 
+    try:
+        dcos_hostname = six.moves.urllib.parse.urlparse(
+            conf.get('core.dcos_url')).hostname
+    except:
+        logger.exception('Unable to find the hostname of the cluster.')
+        dcos_hostname = None
+
     return {
         'cmd': cmd,
         'full_cmd': full_cmd,
         'dcoscli.version': dcoscli.version,
         'python_version': str(sys.version_info),
-        'config': json.dumps(list(conf.property_items()))
+        'config': json.dumps(list(conf.property_items())),
+        'DCOS_HOSTNAME': dcos_hostname,
```

Comment:
I didn't pick the key name. The analytic team want it this way so they can do a `group by`.